cfgov/v1/jinja2/v1/includes/molecules/breadcrumbs.html

```diff
@@ -1,36 +1,50 @@
 {# ==========================================================================
 
-   breadcrumbs.render()
+    breadcrumbs.render()
 
-   ==========================================================================
+    ==========================================================================
 
-   Description:
+    Description:
 
-   Create breadcrumb markup when given:
+    Create breadcrumb markup when given:
 
-   breadcrumbs: An array of tuples used to display nav items.
-                Format: (href, live, title)
+    breadcrumbs: An array of tuples used to display nav items.
+                 Format: (href, live, title)
 
-   ========================================================================== #}
+    ========================================================================== #}
+
+ {% macro render( breadcrumbs ) %}
+     <nav class="m-breadcrumbs" aria-label="{{ _('Breadcrumbs') }}">
+
+         {# Change slash direction in RTL languages. #}
+         {% if language == 'ar' %}
+         {% set crumb_divider = ' \\ ' %}
+         {% else %}
+         {% set crumb_divider = ' / ' %}
+         {% endif %}
+
+         {% for crumb in breadcrumbs %}
+
+            {% if loop.index == loop.length %}
```

Comment:
@anselmbradford in response to your question about the leading slash: in Jinja `for` syntax the `loop.index` variable is 1-based, not 0-based. So if the length of the breadcrumbs is 1, this will render the divider the first time through the loop. See docs [here](https://jinja.palletsprojects.com/en/3.0.x/templates/#for), you can use `loop.index0` instead.

I'm a bit unclear from your comments above with @jenn-franklin -- what is the desired behavior? If you don't want to render the slash if there's only one parent breadcrumb, you could remove this clause, but will it be obvious that this is a breadcrumb then?

Reply:
We didn't include the first slash during usability testing, and participants didn't struggle to understand that it was a breadcrumb, so that's good! But... ugh... I'm waffling. Haha. I guess ultimately I may lean a little more toward including the first slash.

Reply:
As is, http://localhost:8000/es/herramientas-del-consumidor/prestamos-para-vehiculos/respuestas/conozca-sus-derechos/ would not have a leading slash, while http://localhost:8000/es/coronavirus/maneje-sus-finanzas/ would. I guess we either always want the slash or always not have it?

Reply:
Yah, I'd say always slash or never slash.

Reply:
> Yah, I'd say always slash or never slash.

Which one though?

Reply:
If it's helpful, Jinja has a [`random`](https://jinja.palletsprojects.com/en/3.0.x/templates/#jinja-filters.random) filter if we want to show slashes to some visitors but not others. Keep the cf.gov experience dynamic and interesting!

Reply:
Let's go with slash!